<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
background-color: rgb(223, 223, 223);
color: rgb(0, 0, 0);
border-radius: 10px;
}

QComboBox{
background-color: rgb(223, 223, 223);
font: 10pt &quot;Arial&quot;;
color: rgb(0, 0, 0);
border-radius: 10px;
}

QLineEdit:hover{
background-color: rgb(180, 180, 180)
}
QComboBox:hover{
background-color: rgb(180, 180, 180)
}

QPushButton {background-color: rgb(0, 255, 255);
font: 14pt &quot;Arial&quot;;
border-radius: 10px;
border-width: 2px;
hover: rgb(212, 255, 252)
}

QPushButton:hover {
background-color: rgb(100, 255, 255);
}

QPushButton:pressed {
background-color: rgb(27, 198, 186)
}

#cancel {background-color: rgb(255, 80, 80);
font: 14pt &quot;Arial&quot;;
border-radius: 10px;
border-width: 2px;
hover: rgb(212, 255, 252)
}

#cancel:hover {
background-color: rgb(255, 136, 136);
}

#cancel:pressed {
background-color: rgb(186, 3, 3)
}

QLabel{
font: 10pt &quot;Arial&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>530</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>INICIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CANCELAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_ejecutiv">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NOMBRE ANALISTA</string>
    </property>
   </widget>
   <widget class="QLabel" name="campaa">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CAMPAÃ‘A</string>
    </property>
   </widget>
   <widget class="QLabel" name="activity_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ACTIVIDAD</string>
    </property>
   </widget>
   <widget class="QLabel" name="evalua">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ANALISTA EVALUADAOR</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="evaluation_date">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="evaluation_date_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FECHA EVALUACION</string>
    </property>
   </widget>
   <widget class="QComboBox" name="campania">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>389</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="analyst_name">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_form">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <widget class="QComboBox" name="analista_calidad">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="activity">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../media/imagenes/logo.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>HERRAMIENTA DE CALIDAD PARA PT'S TEGGIUM</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="supervisor_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SUPERVISOR@</string>
    </property>
   </widget>
   <widget class="QComboBox" name="supervisor">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="no_siniestro">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_siniestro">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string># DE SINIESTRO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TIPO DE EVALUACION</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tipo_evaluacion">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuEVALUAR_ANALISTA">
    <property name="title">
     <string>EVALUAR ANALISTA</string>
    </property>
   </widget>
   <addaction name="menuEVALUAR_ANALISTA"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
